subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -131,18 +132,32 @@ void ll_iso_tx_mem_release(void *node_tx);
 
 #define NODE_TX_BUFFER_SIZE MROUND(offsetof(struct node_tx_iso, pdu) + \
 				   offsetof(struct pdu_iso, payload) + \
-				   CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE)
+				   MAX(LL_BIS_OCTETS_TX_MAX, \
+				       LL_CIS_OCTETS_TX_MAX))
```

Comment:
When/why can `LL_BIS_OCTETS_TX_MAX` and `LL_CIS_OCTETS_TX_MAX` be different from each other?